.env.development

```diff
@@ -1,5 +1,4 @@
-REACT_APP_DAS_HOST=https://root.dev.pamdas.org
-REACT_APP_GA_TRACKING_ID=UA-128569083-12
```

Comment:
This env variable is not needed anymore?

Reply:
Correct! We transitioned to GA4, so the Universal Analytics ID is no longer needed.